pkg/controller/authentication/authentication_controller.go

```diff
@@ -330,19 +339,27 @@ func updateAuthenticationWithDefaults(authentication *oprv1.Authentication) {
 			authentication.Spec.OIDC.EmailVerification = &defaultVerification
 		}
 	}
+	ldap := authentication.Spec.LDAP
+	if ldap != nil {
+		if ldap.UserSearch.NameAttribute == "" {
+			ldap.UserSearch.NameAttribute = DefaultNameAttribute
+		}
+	}
 }
 
 // validateAuthentication makes sure that the authentication spec is ready for use.
 func validateAuthentication(authentication *oprv1.Authentication) error {
-	// We support using only one connector at once.
-	if authentication.Spec.OIDC != nil && authentication.Spec.Openshift != nil {
-		return fmt.Errorf("multiple identity provider connectors were specified, but only 1 is allowed in the Authentication spec")
-	} else if authentication.Spec.OIDC == nil && authentication.Spec.Openshift == nil {
+	oidc := authentication.Spec.OIDC != nil
+	ldp := authentication.Spec.LDAP
+	numConnectors := render.BoolToInt(oidc) + render.BoolToInt(authentication.Spec.Openshift != nil) + render.BoolToInt(ldp != nil)
```

Comment:
I'm not sure I like this for a couple reasons.
1. oidc is a bool, ldp is a reference, and Openshift doesn't have a short form.
2. The BoolToInt function feels odd as a way to count the "true"s, would this better be served by a more direct function of `func countTrues(b ...bool) int`.

Reply:
Oh I commented on how I thought it was weird to have a  "countTrues" function exported from the utilites, I think "countConnectors" in this function or just having that code inline would be better.